google-cloud-clients/pom.xml

```diff
@@ -157,14 +157,14 @@
     <api-client.version>1.27.0</api-client.version>
     <!-- make sure to keep gax version in sync with google-cloud-bom -->
     <gax.version>1.44.0</gax.version>
-    <api.common.java.version>1.7.0</api.common.java.version>
+    <api.common.java.version>1.8.0</api.common.java.version>
     <google.auth.version>0.15.0</google.auth.version>
     <grpc.version>1.19.0</grpc.version>
     <!-- We are currently using the *-android version to support JDK7. -->
     <guava.version>27.1-android</guava.version>
     <http-client-bom.version>1.29.0</http-client-bom.version>
     <oauth-client.version>1.28.0</oauth-client.version>
-    <protobuf.version>3.7.0</protobuf.version>
+    <protobuf.version>3.7.1</protobuf.version>
```

Comment:
Interesting.. grpc is still on 3.6.1 as of their 1.19.0 release.  grpc 1.20.0 upgraded protobuf to 3.7.1.  I think that we ought to upgrade grpc as well, and probably opencensus to 0.19.2

Reply:
Yes, that makes sense. Not sure any of this is going to work though. There's a dependency-convergence error in google-cloud-core. Probably have to push api-common 1.8.1 and try again to fix that.

Reply:
1.8.1 is out for api-common-java

Reply:
upgraded to 1.8.1; testing now

Reply:
[ERROR] Errors: 
[ERROR] com.google.cloud.datastore.DatastoreTest.com.google.cloud.datastore.DatastoreTest
[ERROR]   Run 1: DatastoreTest.beforeClass:157 » IllegalArgument Invalid version format
[ERROR]   Run 2: DatastoreTest.afterClass:179 » Connect Connection refused (Connection refused)
[INFO] 
[ERROR]   LocalDatastoreHelperTest.testStartStopReset:94 » IllegalArgument Invalid versi...
[INFO] 
[ERROR] Tests run: 168, Failures: 0, Errors: 2, Skipped: 0
[INFO]